Assets/HoloToolkit/Input/Scripts/Utilities/AttachToController.cs

```diff
@@ -11,20 +11,15 @@ namespace HoloToolkit.Unity.InputModule
     /// <summary>
     /// Waits for a controller to be instantiated, then attaches itself to a specified element
     /// </summary>
-    public class AttachToController : MonoBehaviour
+    public class AttachToController : ControllerFinder
     {
-#if  UNITY_WSA && UNITY_2017_2_OR_NEWER
-        public InteractionSourceHandedness Handedness { get { return handedness; } }
-
+#if UNITY_WSA && UNITY_2017_2_OR_NEWER
         [Header("AttachToController Elements")]
         [SerializeField]
-        protected InteractionSourceHandedness handedness = InteractionSourceHandedness.Left;
-
+        protected new InteractionSourceHandedness handedness = InteractionSourceHandedness.Left;
 #endif
-        public MotionControllerInfo.ControllerElementEnum Element { get { return element; } }
-
         [SerializeField]
-        protected MotionControllerInfo.ControllerElementEnum element = MotionControllerInfo.ControllerElementEnum.PointingPose;
+        protected new MotionControllerInfo.ControllerElementEnum element = MotionControllerInfo.ControllerElementEnum.PointingPose;
```

Comment:
Why the new keyword?

Reply:
*AttachToController* is overriding the The base class *ControllerFinder*'s protected element variable to be a SerializeField. Unity requires the *new* keyword when hiding the base class variable.